recipes/r-pegboard/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set version = '0.7.9' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-pegboard
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/carpentries/pegboard/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ee9c429e792817ea269b43c98bc95bcab62aaff41d30754377cd4d7e73539611
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-r6
+    - r-commonmark
+    - r-fs >=1.5.0
+    - r-glue
+    - r-purrr
+    - r-tinkr >=0.2.0
+    - r-xml2
+    - r-xslt
+    - r-yaml
+  run:
+    - r-base
+    - r-r6
+    - r-commonmark
+    - r-fs >=1.5.0
+    - r-glue
+    - r-purrr
+    - r-tinkr >=0.2.0
```

```suggestion
    - r-tinkr >=0.3.0
    - r-xml2
    - r-xslt
    - r-yaml
  run:
    - r-base
    - r-r6
    - r-commonmark
    - r-fs >=1.5.0
    - r-glue
    - r-purrr
    - r-tinkr >=0.3.0
```

Given discussion in https://github.com/conda-forge/r-tinkr-feedstock/issues/4 the failing unit tests will get fixed in `r-tinkr` `v0.3.0` (c.f. https://github.com/ropensci/tinkr/issues/134) and so the required `r-tinkr` version should get bumped.

---

This can go in for new releases of `pegboard` later.